src/storage/src/hummock/compactor/mod.rs

```diff
@@ -204,27 +208,14 @@ impl Compactor {
         Ok((ssts, table_stats_map))
     }
 
-    pub async fn report_progress(
+    pub fn report_progress(
         metrics: Arc<CompactorMetrics>,
         task_progress: Option<Arc<TaskProgress>>,
-        split_table_outputs: Vec<SplitTableOutput>,
+        ssts: &Vec<LocalSstableInfo>,
```

Comment:
What benefit can we get if we pass `&Vec<LocalSstableInfo>` instead of `Vec<SplitTableOutput>` here? After this PR, we need to create a new vec via `collect` every time before calling `report_progress`, which seems like burden and overhead that can be avoided.

Reply:
Previously, when report_progress was called, the vec was built from scratch and the LocalSstableInfo was taken from the SplitTableOutput, which is similar.